differs/pipDiff.go

```diff
@@ -37,27 +39,50 @@ func getPythonVersion(pathToLayer string) ([]string, error) {
 	return matches, nil
 }
 
-func (d PipDiffer) getPackages(path string) (map[string]map[string]utils.PackageInfo, error) {
-	packages := make(map[string]map[string]utils.PackageInfo)
+func getPythonPaths(vars []string) []string {
```

Comment:
To clarify, is every path where python packages could be installed included in the environment variables?  Or is it only the path set on exit?